src/onnx/parse_dynamicquantizelinear.cpp

```diff
@@ -131,8 +134,8 @@ struct parse_dynamicquantizelinear : op_parser<parse_dynamicquantizelinear>
         // intermediate_zero_point = qmin - min(x) / y_scale
         auto q_min     = info.add_literal(migraphx::literal{migraphx::shape{x_type}, {x_min}});
         auto q_max     = info.add_literal(migraphx::literal{migraphx::shape{x_type}, {x_max}});
-        auto sub1      = info.add_common_op("sub", q_min, min_x);
-        auto interm_zp = info.add_common_op("div", sub1, y_scale);
+        auto div1      = info.add_common_op("div", min_x, y_scale);
```

Comment:
The variables x_min & min_x are very confusing.. (same for x_max & max_x).. One set  could be renamed to [t_max & t_min] the other to maybe [cur_max & cur_min].. ? Please choose some names you like..

Reply:
Done